SwiftUI_Homework/SwiftUI_Homework/PlayerView.swift

```diff
@@ -0,0 +1,252 @@
+//
+//  PlayerView.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 16.01.2023.
+//
+
+import SwiftUI
+import AVKit
+
+/// Экран "Плеер" c navigationTitle
+struct PlayerView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let playerText = "Music Player"
+    }
+    
+    var body: some View {
```

Comment:
```suggestion

// MARK: - Body
    var body: some View {
```
наверное тоже должна быть марка какая-то